lib/modules/datasource/metadata.ts

```diff
@@ -170,3 +171,31 @@ export function addMetaData(
     }
   }
 }
+
+// remove homepage if both sourceUrl and homepage exists and:
+// 1. homepage its not a link to a repository subpath
```

Comment:
i think this isn't true, please update the docs

Reply:
Is this more clear ? 
```suggestion
// 1. homepage is not a link to a path in repo (e.g https://github.com/org/repo/)
```
I think if the function's code is clear enough we can remove the comment above it entirely.